integration/data/conftest.py

```diff
@@ -49,6 +50,38 @@ def grpc_controller_no_frontend(request, grpc_controller_client):
                                   frontend="")
 
 
+@pytest.fixture
+def first_available_device(request):
+    # we find the next available scsi name based on the current
+    # nodes in the /dev filesystem which isn't fool proof
+    device_prefix = "/dev/sd"
+    first_available_device = "does_not_exist"
```

Comment:
Can do `= ""`

Reply:
I did this on purpose, so that if for whatever reason we cannot find an available device we don't try creating a link of "" to /dev/null